app/Controllers/Event.php

```diff
@@ -23,6 +23,7 @@ public function get(int|null $year = null) {
         $speakers = $speakerModel->getPublished($event['id']);
 
         return $this->response->setJSON([
+            'year' => $year,
```

Comment:
I would expect the year inside the event.
Why did you put it directly inside the main json?

Reply:
I just thought that this is the simplest way of implementing it. Do you want me to change it? Shouldn't be difficult.

Reply:
The difference in the frontend is the type where its located after parsing.
https://github.com/TechStreamConference/website-frontend/blob/main/src/types/provideTypes.ts
Its ether inside the event or the Year.
But I think inside the Event would be not bad for the backend later on.

Reply:
Changed it. Take a look :)